tests/queries/0_stateless/03254_timeseries_functions.reference

```diff
@@ -0,0 +1,11 @@
+[(1734955421.374,0),(1734955436.374,0),(1734955451.374,0),(1734955466.374,0),(1734955481.374,0),(1734955496.374,0),(1734955511.374,1),(1734955526.374,3),(1734955541.374,5),(1734955556.374,5),(1734955571.374,5),(1734955586.374,5),(1734955601.374,8),(1734955616.374,8),(1734955631.374,8),(1734955646.374,8),(1734955661.374,8),(1734955676.374,8)]
+[(1734955380,NULL),(1734955395,NULL),(1734955410,NULL),(1734955425,0),(1734955440,0),(1734955455,0),(1734955470,0),(1734955485,0),(1734955500,0),(1734955515,1),(1734955530,3),(1734955545,5),(1734955560,5),(1734955575,5),(1734955590,5),(1734955605,8),(1734955620,8),(1734955635,8),(1734955650,8),(1734955665,8),(1734955680,8)]
+Row 1:
+──────
+rate_5m:  [(1734955380,NULL),(1734955395,NULL),(1734955410,NULL),(1734955425,NULL),(1734955440,0),(1734955455,0),(1734955470,0),(1734955485,0),(1734955500,0),(1734955515,0.003467629629629629),(1734955530,0.010345333333333333),(1734955545,0.017170277777777774),(1734955560,0.017114320987654322),(1734955575,0.017069555555555557),(1734955590,0.01703292929292929),(1734955605,0.027203851851851854),(1734955620,0.02716252991452992),(1734955635,0.027127111111111112),(1734955650,0.02709641481481482),(1734955665,0.02706955555555555),(1734955680,0.02704585620915033)]
```

Comment:
I tried to check value `(1734955515,0.003467629629629629)` here.
The window was 300 s, so the first and the last points are `(1734955421.374,0)` and `(1734955511.374,1)`. Thus it seems the result should be `(1 - 0) / (1734955511.374 - 1734955421.374)` which is `0.011111111` and not `0.003467629629629629`. Did I miss something?